docs/en/compute/AWS-EMR.md

```diff
@@ -28,59 +28,72 @@ different cloud provider's storage i.e. GCS, Azure Blob Store.
 * AWS CLI: Make sure that the AWS CLI is also set up and ready with the required AWS Access/Secret key
 
 The majority of the pre-requisites can be found by going through the
-[AWS EMR Getting Started](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html) guide. An S3 bucket
-is needed as Alluxio's Root Under File System and to serve as the location for the bootstrap script. If required,
-the root UFS can be reconfigured to be HDFS.
+[AWS EMR Getting Started](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html) guide.
+An S3 bucket is needed as Alluxio's Root Under File System and to serve as the location for the
+bootstrap script.
+If required, the root UFS can be reconfigured to be HDFS.
 
 ## Basic Setup
 
 To begin with, [download an Alluxio release](https://www.alluxio.io/download) and unzip it.
 
 1. Set up the required IAM roles for the account to be able to use the EMR service.
-```bash
-aws emr create-default-roles
+```console
+$ aws emr create-default-roles
 ```
 2. Download `alluxio-emr.sh` from [our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/) and
 upload the script to your S3 bucket.
-3. The input arguments for the bootstrap script are in the order below:
-    - The URI from where to download the Alluxio Release `.tar.gz` file. This can be an `http://`
-     URI, `s3://` URI or an `http://` URI.
-     This is a mandatory property.
-    - The root-ufs-uri. This should be an `s3://` or `hdfs://` URI designating the root mount of the Alluxio file system.
-       This is a mandatory property.
-    - Extra alluxio options. These are specified as a comma-separated list of key-values in the format `<key>=<value>`.
-       For example, `alluxio.user.file.writetype.default=CACHE_THROUGH`
-```
-aws emr create-cluster \
+3. There are 5 input arguments for the bootstrap script. They are described below:
+    - (Mandatory) The URI from where to download the Alluxio Release `.tar.gz` file.
+    This can be an `https://` URI, `s3://` URI or an `http://` URI.
+    - (Mandatory) The root-ufs-uri.
+      This should be an `s3://` or `hdfs://` URI designating the root mount of the Alluxio file
```

Comment:
why is this required to be s3 or hdfs? @ns1123

Reply:
Ping @ns1123  this is the Alluxio UFS URI and can be any UFS alluxio supports, right?